core/templates/components/state-editor/state-solution-editor/state-solution-editor.component.ts

```diff
@@ -131,8 +131,17 @@ export class StateSolutionEditorComponent implements OnInit {
 
   getSolutionSummary(): string {
     let solution = this.stateSolutionService.savedMemento;
-    let solutionAsPlainText = (
-      solution.getSummary(this.stateInteractionIdService.savedMemento));
+    let interactionId = this.stateInteractionIdService.savedMemento;
+    let solutionAsPlainText = '';
+    if (interactionId === 'DragAndDropSortInput') {
```

Comment:
Conceptually, solution editor should work as intended without having to know the kind of interaction it is dealing with. Note that we don't have interaction id checks elsewhere in this component. Either pass the customisationArgs for all interactions since the interaction id specific formatting is done in the solution object or introduce a property in INTERACTION_SPECS and check that here.

Reply:
Got it, I've modified it so the customization args are passed in for all interactions.